modules/ROOT/pages/queries/select-version.adoc

```diff
@@ -0,0 +1,251 @@
+
+Below are two examples showing the two versions of Cypher behaving differently.
+
+.Differences in setting properties between Cypher 5 and Cypher 25
+======
+The first query uses a `RELATIONSHIP` instead of `MAP` on the right-hand side of a xref:clauses/set.adoc[`SET`] clause.
+This was allowed in Cypher 5, but disallowed in Cypher 25, which requires that the xref:functions/scalar.adoc#functions-properties[`properties()`] function is used to get the map of properties from nodes or relationships before referencing them in a `SET` clause.
+
+.Setting properties on a node from a relationship using Cypher 5
+[source, cypher]
+----
+CYPHER 5
+MATCH (n:Order)-[r:SHIPPED_TO]->(:Address) 
+SET n = r
+----
+
+.Setting properties on a node from a relationship using Cypher 25
+[source, cypher]
+----
+CYPHER 25
+MATCH (n:Order)-[r:SHIPPED_TO]->(m:Address) 
+SET n = properties(r)
+----
+======
+
+.Differences in listing constraints between Cypher 5 and Cypher 25
+======
+In this example, the same statement listing the `name` and `type` of the xref:constraints/index.adoc[constraints] in a database is executed first using Cypher 5 and then with Cypher 25.
+Note that the return values in the `type` column have been updated in the results from the statement run with Cypher 25.
+
+////
+CREATE CONSTRAINT node_uniqueness_constraint
+FOR (n:Label) REQUIRE n.property IS UNIQUE;
+CREATE CONSTRAINT rel_uniqueness_constraint
+FOR ()-[r:REL_TYPE]-() REQUIRE r.property IS UNIQUE;
+////
+
+.Listing the constraint name and type values using Cypher 5
+[source, cypher]
+----
+CYPHER 5
+SHOW CONSTRAINTS YIELD name, type
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| name | type 
+
+| "node_uniqueness_constraint" | "UNIQUENESS"
+| "rel_uniqueness_constraint"  | "RELATIONSHIP_UNIQUENESS"
+
+2+d|Rows: 2
+|===
+
+.Listing the constraint name and type values using Cypher 25
+[source, cypher]
+----
+CYPHER 25
+SHOW CONSTRAINTS YIELD name, type
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| name | type 
+
+| "node_uniqueness_constraint" | "NODE_PROPERTY_UNIQUENESS"
+| "rel_uniqueness_constraint"  | "RELATIONSHIP_PROPERTY_UNIQUENESS"
+
+2+d|Rows: 2
+|===
+
+======
+
+If a database has specified a default Cypher version, it is possible to set a different Cypher version for individual queries.
+For example, if a database is created with Cypher 5 as its default language, prepending a query run on that database with `CYPHER 25` ensures the query uses Cypher 25.
+
+[[procedures-and-functions]]
+=== Procedures and functions
+
+link:{neo4j-docs-base-uri}/operations-manual/current/procedures[Procedures] and xref:functions/index.adoc[functions] (including built-in and link:{neo4j-docs-base-uri}/apoc/current/[APOC]) are tied to a specific Cypher language version.
```

Comment:
why does `Procedures` have a capital P but `functions` not a capital F?

Reply:
Because "Procedures" is the first worst of the sentence.

Reply:
Ah, fair. Missed that as the link was so long before it and I only looked in the code and not the built page XD